docs/examples/openstack-credential-v3.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: Secret
+metadata:
+# this is the jenkins id.
+  name: "test-openstack-credential-v3"
+  labels:
+# so we know what type it is.
+    "jenkins.io/credentials-type": "openstackCredentialv3"
+  annotations:
+# description - can not be a label as spaces are not allowed
+    "jenkins.io/credentials-description" : "openstack credentials for you Jenkins in Kubernetes!"
+type: Opaque
+stringData:
+  userName: Y2FzdWFsTmFtZQ==               #casualName
```

Comment:
these shouldn't be base64 encoded as they are in a `stringData` not `string`

Reply:
decrypted all records